site/src/app/components/verdictPanel/styles.module.scss

```
@@ -1,9 +1,12 @@
-.container {
-  border-radius: 16px 16px 16px 0px;
+.background {
   background-color: var(--color-white);
-  padding: 1rem;
+}
+
+.bordered{
+  border-radius: 16px 16px 16px 0px;
 }
 
 .callout-title {
   margin-bottom: 1rem;
+  font-size: 24px;
```

Il faudrait utiliser l'unité de mesure REM pour les font-size idéalement pour l'accessibilité plus tard
(24/16) => 1.5

```suggestion
  font-size: 1.5rem;
```

---

done